.github/workflows/buildAndTest.yml

```diff
@@ -8,13 +8,6 @@ on:
     types: [assigned, opened, synchronize, reopened]
   workflow_dispatch:
 
-defaults:
-  run:
-    # This is already the default, except when running inside another Docker
-    # image, which is the case here. So set it up globally to avoid
-    # repeating elsewhere.
-    shell: bash
-
```

Comment:
Out of curiosity, is this no longer true? Was it never true?

Reply:
Because I removed the containerization and so it is indeed now the default.